### PR TITLE
Adds the help command

### PR DESCRIPTION
**Pull request by PowerSchill**

### CODE REVIEW

**Comment by ravenbuilder934** [comment]
Why did you change the table? It's already centered: 
Annotation 2020-06-14 103936

**Comment by PowerSchill** [comment]
I think it was just Visual Studio Code formatting the document on save.

File: source/systems/commands.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ These are the in game commands currently available, most require that a user hav
       <td class="d-block d-lg-table-cell"><code>/mc scan [player: online player] &lt;x1: x coord&gt; &lt;y1: y coord&gt; &lt;z1: z coord&gt; &lt;x2: x coord&gt; &lt;y2: y coord&gt; &lt;z2: z coord&gt; [name: name of schematic]</code><br><br><span class="perm">Permission node: com.minecolonies.coremod.Scan</span><br>Default Permission Level: OP</td>
       <td class="d-block d-lg-table-cell">Scans an area and saves it as a schematic (Command version of what the scan tool can be used for)<br><br>Examples:<br><code>/mc scan x1: 1 y1: 1 z1: 1 x2: 5 y2: 5 z2: 5</code><br><code>/mc scan player: Steve x1: 1 y1: 1 z1: 1 x2: 5 y2: 5 z2: 5</code><br><code>/mc scan x1: 1 y1: 1 z1: 1 x2: 5 y2: 5 z2: 5 name: MyAwesomeDecoration</code></td>
     </tr>
+     <tr class="d-block d-lg-table-row">
+      <td class="d-block d-lg-table-cell"><code>/mc help</code><br><br><span class="perm">Permission node: com.minecolonies.coremod.Scan</span><br>Default Permission Level: ALL</td>
+      <td class="d-block d-lg-table-cell">Displays the URL for this wiki as well as the URL for the Discord server.</code></td>
```
Comment (ravenbuilder934) [inline source/systems/commands.md:78]:
```suggestion
      <td class="d-block d-lg-table-cell">Displays the URL for the wiki as well as the URL for the Discord server.</code></td>
```

**Comment by lunadragnstar** [comment]
You only added the help command? Didn't change any of the other commands? Sorry just had to tease you a bit.

**Comment by lunadragnstar** [comment]
I'm going to cancel this one as I have to fix all the other commands.

**Pull request closed without merging**